src/components/material/material-buttons/physical/MaterialButtonsPhysical.tsx

```diff
@@ -25,27 +27,33 @@ const MaterialButtonsPhysical: React.FC<MaterialButtonsPhysicalProps> = ({
   const { reservableManifestations } = UseReservableManifestations({
     manifestations
   });
+  const { data: userData, isLoading } = useGetPatronInformationByPatronIdV2();
 
-  // TODO: use useGetPatronInformationByPatronIdV2() when we get the correctly
-  // set up STORYBOOK_CLIENT_ID from Rolf. The "isUserBlocked" is temporary.
-  const isUserBlocked = false;
-
-  if (isUserBlocked) {
-    return <MaterialButtonUserBlocked size={size} />;
+  if (isLoading) {
+    return <MaterialButtonLoading />;
   }
 
   if (!reservableManifestations || reservableManifestations.length < 1) {
     return <MaterialButtonCantReserve size={size} />;
   }
 
-  return (
-    <MaterialButtonReservePhysical
-      dataCy={dataCy}
-      manifestationMaterialType={getManifestationType(manifestations)}
-      faustIds={faustIds}
-      size={size}
-    />
-  );
+  if (userData?.patron?.blockStatus) {
+    return <MaterialButtonUserBlocked size={size} dataCy={dataCy} />;
+  }
+
+  // We show the reservation button if either
+  if (!userData || !userData?.patron?.blockStatus) {
+    return (
+      <MaterialButtonReservePhysical
```

Comment:
Consider explaining *why* it is important to show the reservation button in these two cases in the code comment.

The current form merely repeats the contents of the `if` statement and is not really that helpful. I guess it has something to do with the reservation-button redirecting to login if the user is not authenticated and thus no user data is available?